docker/aeternity_node_mean16.yaml

```diff
@@ -20,7 +20,7 @@ chain:
   persist: false
   hard_forks:
     "1": 0
-    "5": 1
```

Comment:
@davidyuk How these are numbered?

Reply:
the key is a version of consensus protocol and the value is the height at which protocol should be applied, so I'm switching our node for integration tests from 5th propcol to 6th, it is strange that nothing broke

Reply:
Nice, Thanks.

Reply:
> it is strange that nothing broke

that means the APIs are backward compatible and most of the consensus I guess

Reply:
also keep in mind that HC branch and `v0.0.0-hc5` respectively somehow behind master